Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -452,40 +452,76 @@ void doPublishRawImu(void) {
  * Publish bumper data to ros.
  */
 void doPublishBumpers(void) {
-  // TODO(mwomack): Get readings from robot
-  static const int MIN_RAW_LEFT_BUMPER(650);
-  static const int MAX_RAW_LEFT_BUMPER(450);
-  static const double RANGE_RAW_LEFT_BUMPER(MAX_RAW_LEFT_BUMPER - MIN_RAW_LEFT_BUMPER);
-  static const double MAX_FORCE_LEFT_BUMPER(1.0);
-  static const int MIN_RAW_RIGHT_BUMPER(650);
-  static const int MAX_RAW_RIGHT_BUMPER(450);
-  static const double RANGE_RAW_RIGHT_BUMPER(MAX_RAW_RIGHT_BUMPER - MIN_RAW_RIGHT_BUMPER);
-  static const double MAX_FORCE_RIGHT_BUMPER(1.0);
+  // These values were read as samples from the robot and collected here
+  // to return a distance value for any sensor reading from the robot.
+  // Each sensor sample has a distance sample associated with it. For sensor
+  // readings that fall between sampled values, a linear approximation is
```

Comment:
Mention that the arrays are arranged from lowest to highest, otherwise it would break.